runtime/src/accounts_db.rs

```diff
@@ -1946,10 +1947,14 @@ impl AccountsDb {
                 }
                 start = next;
             }
+            num_stores += 1;
         }
 
         let mut index_read_elapsed = Measure::start("index_read_elapsed");
         let mut alive_total = 0;
+        let mut alive_accounts = Vec::with_capacity(stored_accounts.len() / 2);
```

Comment:
Why is this len() / 2? Do we expect 1/2 the accounts to be alive?

Reply:
Seemed like a decent guess for it